cmd/cosign/cli/signblob.go

```diff
@@ -41,6 +41,9 @@ func SignBlob() *cobra.Command {
   # sign a blob with a local key pair file
   cosign sign-blob --key cosign.key <FILE>
 
+  # sign a blob with local certificate and CA bundle
+  cosign sign-blob --key cosign.key --certificate <FILE> --certificate-chain <FILE> <FILE>
```

Comment:
Should this example show outputting the bundle too, since that is the main use case for adding these?

Reply:
Ah yes good point!